integreat_cms/cms/templates/languages/region_list_section/region_list.html

```diff
@@ -0,0 +1,31 @@
+{% load i18n %}
+{% load static %}
+{% load content_filters %}
+{% block content %}
+    <div class="table-listing">
+        <table class="w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white text-center">
+            <thead>
+                {% get_language_active_regions language as active_in_regions %}
+                {% get_language_visible_regions language as visible_in_regions %}
+                {% get_language_active_visible_regions language as active_and_visible_in_regions %}
+                <tr class="border-b border-solid border-gray-200">
+                    <th class="text-sm text-left uppercase py-3 pl-4 pr-2">{% translate "Region" %}</th>
+                    <th class="text-sm text-left uppercase py-3 px-2">{% translate "Active" %} ({{ active_in_regions|length }})</th>
+                    <th class="text-sm text-left uppercase py-3 px-2">{% translate "Visible" %} ({{ visible_in_regions|length }})</th>
```

Comment:
```suggestion
                <tr class="border-b border-solid border-gray-200">
                    <th class="text-sm text-left uppercase py-3 pl-4 pr-2">{% translate "Region" %}</th>
                    <th class="text-sm text-left uppercase py-3 px-2">{% translate "Active" %}</th>
                    <th class="text-sm text-left uppercase py-3 px-2">{% translate "Visible" %}</th>
```
I think it would be fine to remove the template tags (and also loose the possibility to show the active count)... If you want to show it anyways, I would add a new property to the language model

```
    @cached_property
    def active_in_regions(self):
        return Region.objects.filter(
            language_tree_nodes__language=self, language_tree_nodes__active=True
        )

    @cached_property
    def visible_in_regions(self):
        return Region.objects.filter(
            language_tree_nodes__language=self,
            language_tree_nodes__active=True,
            language_tree_nodes__visible=True,
        )
```